### PR TITLE
style: config item for each rule

### PR DESCRIPTION
**Pull request by nnneeennn**
### This PR:

(If this is a release PR, no need to add following. Leave this part empty)
(Use the following lines to create a PR text body. Make sure to remove all non-relevant one after you're done)
(Repeat each field as many times as necessary)

Added: for new features.
Changed: for changes in existing functionality.
Deprecated: for soon-to-be removed features.
Removed: for now removed features.
Fixed: for any bug fixes.
Security: in case of vulnerabilities.

### Review and tests:
- [ ] Tests pass
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Code review
- [ ] `CHANGELOG.md` is updated
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by maehler** [comment]
Thanks for having a look at this! The reason I made it like this was that these are config parameters that affect how the final report would appear. I agree that it doesn't match with how the code is organised, and what parameters go with which rule, but I'm not sure it is the best approach. For example, the `show_table` parameter is directly connected with what will show up in the report, but it should also affect what data is kept when merging the JSON files. If it goes under `merge_cnv_json` it should probably be named differently.

In short, I agree with you, and disagree with you. 😅 I'll have to think a bit about it, but it will probably end up being closer to your suggestion that the original code.

**Comment by nnneeennn** [comment]
Ah I see! I only saw that none of the other rules are obviously using those files as input or params and thought the structure of the config file should reflect that. Given that we can also just import a single rule from hydra . Which in this case does not make any sense, but you know, for keeping things within the style

**Comment by nnneeennn** [comment]
but also feel free to just archive the branch if not needed

**Comment by maehler** [comment]
I agree that it would be nice to keep the style consistent. Thinking about it some more, I think it would make sense to have the `show_table` parameter under `cnv_html_report`, but let the other parameters remain under `merge_cnv_json`, as you suggest. I could still make the conditional requirement of the VCF files work with this. What do you think about that?

**Comment by nnneeennn** [comment]
that sounds great
